Mathlib/Geometry/Convex/Cone/Pointed.lean

```diff
@@ -63,7 +63,7 @@ instance instZero (C : PointedCone R E) : Zero C :=
   ⟨0, C.zero_mem⟩
 
 /-- The `PointedCone` constructed from a pointed `ConvexCone`. -/
-def _root_.ConvexCone.toPointedCone {C : ConvexCone R E} (hC : C.Pointed) : PointedCone R E where
+def _root_.ConvexCone.toPointedCone (C : ConvexCone R E) (hC : C.Pointed) : PointedCone R E where
```

Comment:
Isn't it more helpful still to add zero to the convex cone to make it pointed, then you don't need to take hC at all?

Reply:
Maybe? but the issue is that it changes the defeqs. I am not against it, but this is a bigger change whose ramifications I would need to investigate.

Reply:
Fair enough